partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -102,6 +114,11 @@ rex::{
       symbol::{ root: rex, key: string },
     ],
 
+    cast_op::{
+      arg: rex,
+      cast: cast
+    },
+
```

Comment:
See modeling in https://github.com/partiql/partiql-lang-kotlin/blob/partiql-spi-planner/partiql-planner/src/main/resources/partiql_plan_internal.ion/